### PR TITLE
init model

### PR DESCRIPTION
**Pull request by vietbq**

### CODE REVIEW

File: app/models/user_book.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UserBook < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :book
+
```
Comment (hellovietnam93) [inline app/models/user_book.rb]: remove this line

File: app/models/review.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :book
+  has_many :comments
+
```
Comment (hellovietnam93) [inline app/models/review.rb]: remove thís line

File: app/models/review.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :book
+  has_many :comments
```
Comment (hellovietnam93) [inline app/models/review.rb]: add new line to divide as a block

File: app/models/request.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class Request < ActiveRecord::Base
+  belongs_to :user
+
+  validates :book_name, presence: true
+  validates :content, presence: true
+  validates :author, presence: true
+
```
Comment (hellovietnam93) [inline app/models/request.rb]: remove this line

**Comment by hellovietnam93** [comment]
OK

File: app/models/relationship.rb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+class Relationship < ActiveRecord::Base
+  belongs_to :follower, class_name: "User"
```
Comment (dieunb) [inline app/models/relationship.rb]: `class_name: User.name`

File: app/models/user.rb [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable,
+    :trackable, :validatable
+
+  has_many :user_books
+  has_many :favorites
+  has_many :reviews
+  has_many :comments
+  has_many :requests
+  has_many :likes
+  has_many :active_relationships, class_name:  "Relationship",
```
Comment (dieunb) [inline app/models/user.rb]:
```
+  has_many :active_relationships, class_name:  Relationship.name,
 +    foreign_key: :follower_id, dependent:   :destroy
```

**Pull request merged**